display_list/display_list_utils.cc

```diff
@@ -256,13 +258,17 @@ void DisplayListBoundsCalculator::setStrokeWidth(SkScalar width) {
 void DisplayListBoundsCalculator::setStrokeMiter(SkScalar limit) {
   miter_limit_ = std::max(limit, 1.0f);
 }
-void DisplayListBoundsCalculator::setBlendMode(SkBlendMode mode) {
+void DisplayListBoundsCalculator::setBlendMode(DlBlendMode mode) {
   blend_mode_ = mode;
 }
 void DisplayListBoundsCalculator::setBlender(sk_sp<SkBlender> blender) {
   SkPaint paint;
   paint.setBlender(std::move(blender));
-  blend_mode_ = paint.asBlendMode();
+  blend_mode_ = std::nullopt;
```

Comment:
Why not wait until you've tested the paint's mode and do this in an "else" case below?